bin/cron/osh-remove-empty-folders.sh

```diff
@@ -12,45 +12,11 @@ basedir=$(readlink -f "$(dirname "$0")"/../..)
 # shellcheck source=lib/shell/functions.inc
 . "$basedir"/lib/shell/functions.inc
 
-trap "_err 'Unexpected termination!'" EXIT
-
-# setting default values
-LOGFILE=""
-LOG_FACILITY="local6"
-ENABLED=1
+# default config values for this script
 MTIME_DAYS=1
 
-# building config files list
-config_list=''
-if [ -f "$BASTION_ETC_DIR/osh-remove-empty-folders.conf" ]; then
-    config_list="$BASTION_ETC_DIR/osh-remove-empty-folders.conf"
-fi
-if [ -d "$BASTION_ETC_DIR/osh-remove-empty-folders.conf.d" ]; then
-    config_list="$config_list $(find "$BASTION_ETC_DIR/osh-remove-empty-folders.conf.d" -mindepth 1 -maxdepth 1 -type f -name "*.conf" | sort)"
-fi
-
-if [ -z "$config_list" ]; then
-    exit_fail "No configuration loaded, aborting"
-fi
-
-# load the config files only if they're owned by root:root and mode is o-rwx
-for file in $config_list; do
-    if check_secure "$file"; then
-        # shellcheck source=etc/bastion/osh-remove-empty-folders.conf.dist
-        . "$file"
-    else
-        exit_fail "Configuration file not secure ($file), aborting."
-    fi
-done
-
-# shellcheck disable=SC2153
-if [ -n "$LOGFILE" ] ; then
-    exec &>> >(tee -a "$LOGFILE")
-fi
-
-if [ "$ENABLED" != 1 ]; then
-    exit_success "Script is disabled"
-fi
+# set error trap, read config, setup logging, exit early if script is disabled, etc.
+script_init osh-remove-empty-folders config_optional check_secure_lax
```

Comment:
Before:  check_secure 
After: check_secure_lax

Wanted ?

Reply:
Yes: the config file of this script doesn't bear any credentials, so there's no need for it to be `o-r`, it should still be `o-w` though (lax)